### PR TITLE
Update mddocs for DockerGuides

### PR DESCRIPTION
**Pull request by ATMxsp01**
## Description

https://github.com/analytics-zoo/nano/issues/1457

Update part of Quickstart guide in mddocs

Preview: https://github.com/ATMxsp01/ipex-llm/tree/rst2md/docs/mddocs/DockerGuides

### CODE REVIEW

File: docs/mddocs/DockerGuides/docker_cpp_xpu_quickstart.md [hunk 1]
```diff
@@ -24,18 +24,18 @@ docker pull intelanalytics/ipex-llm-inference-cpp-xpu:latest
...
 ### Start Docker Container
 
-```eval_rst
-.. tabs::
-   .. tab:: Linux
+Choose one of the following methods to start the container:
 
-      To map the `xpu` into the container, you need to specify `--device=/dev/dri` when booting the container. Select the device you are running(device type:(Max, Flex, Arc, iGPU)). And change the `/path/to/models` to mount the models. `bench_model` is used to benchmark quickly. If want to benchmark, make sure it on the `/path/to/models`
+<details>
+<Summary>For <strong>Linux</strong>:</summary>
```
Comment (JinBridger) [inline docs/mddocs/DockerGuides/docker_cpp_xpu_quickstart.md:30]: better to use bullet point here?

File: docs/mddocs/DockerGuides/docker_cpp_xpu_quickstart.md [hunk 2]
```diff
@@ -69,9 +71,10 @@ docker pull intelanalytics/ipex-llm-inference-cpp-xpu:latest
                 -e DEVICE=Arc \
                 --shm-size="16g" \
                 $DOCKER_IMAGE
+  ```
+</details>
 
-```
-
+---
```
Comment (JinBridger) [inline docs/mddocs/DockerGuides/docker_cpp_xpu_quickstart.md:77]: better to remove this dividing line?

File: docs/mddocs/DockerGuides/docker_pytorch_inference_gpu.md [hunk 3]
```diff
@@ -20,37 +16,37 @@ Prepare ipex-llm-xpu Docker Image:
...
-Start ipex-llm-xpu Docker Container:
-
-```eval_rst
-.. tabs::
-   .. tab:: Linux
+Start ipex-llm-xpu Docker Container. Choose one of the following commands to start the container:
 
-      .. code-block:: bash
+<details>
+<summary>For <strong>Linux</strong>:</summary>
```
Comment (JinBridger) [inline docs/mddocs/DockerGuides/docker_pytorch_inference_gpu.md:22]: better to use bullet point here?

File: docs/mddocs/DockerGuides/docker_pytorch_inference_gpu.md [hunk 4]
```diff
@@ -60,8 +56,10 @@ Start ipex-llm-xpu Docker Container:
                 -v $MODEL_PATH:/llm/llm-models \
                 -v /usr/lib/wsl:/usr/lib/wsl \ 
                 $DOCKER_IMAGE
-```
+  ```
+</details>
 
+---
```
Comment (JinBridger) [inline docs/mddocs/DockerGuides/docker_pytorch_inference_gpu.md:62]: better to remove this dividing line?

File: docs/mddocs/DockerGuides/docker_run_pytorch_inference_in_vscode.md [hunk 5]
```diff
@@ -52,37 +49,38 @@ Open the Terminal in VSCode (you can use the shortcut `` Ctrl+Shift+` ``), then
...
 ```
 
-Start ipex-llm-xpu Docker Container:
+Start ipex-llm-xpu Docker Container. Choose one of the following commands to start the container:
 
-```eval_rst
-.. tabs::
-   .. tab:: Linux
+<details>
+<summary>For <strong>Linux</strong>:</summary>
```
Comment (JinBridger) [inline docs/mddocs/DockerGuides/docker_run_pytorch_inference_in_vscode.md:55]: better to use bullet point here?

File: docs/mddocs/DockerGuides/docker_run_pytorch_inference_in_vscode.md [hunk 6]
```diff
@@ -92,8 +90,10 @@ Start ipex-llm-xpu Docker Container:
                 -v $MODEL_PATH:/llm/llm-models \
                 -v /usr/lib/wsl:/usr/lib/wsl \ 
                 $DOCKER_IMAGE
-```
+  ```
+</details>
 
+---
```
Comment (JinBridger) [inline docs/mddocs/DockerGuides/docker_run_pytorch_inference_in_vscode.md:96]: better to remove this dividing line?

File: docs/mddocs/DockerGuides/docker_cpp_xpu_quickstart.md [hunk 7]
```diff
@@ -194,13 +197,17 @@ Sample output:
...
+Please refer to this [documentation](../Quickstart/ollama_quickstart.md#4-pull-model) for more details.
 
 
 ### Running Open WebUI with Intel GPU
 
 Start the ollama and load the model first, then use the open-webui to chat.
-If you have difficulty accessing the huggingface repositories, you may use a mirror, e.g. add export HF_ENDPOINT=https://hf-mirror.com before running bash start.sh.
+If you have difficulty accessing the huggingface repositories, you may use a mirror, e.g. add 
+```bash
+export HF_ENDPOINT=https://hf-mirror.com
```
Comment (JinBridger) [inline docs/mddocs/DockerGuides/docker_cpp_xpu_quickstart.md:208]: better to use inline code here?

**Review by Oscilloscope98 — APPROVED** [review]
Merge for now, May add further style change

**Pull request merged**